index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Version Skeleton</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main class="maincontainer">
+        <header>
+            <a href="#" class="logo">Crysty</a>
+            <a href="#" class="menu"><img src="images/Union.png" alt="dropdown menu"> </a>
+        </header>
+        <section class="infos">
+
+            <h1>I’m Christine Glad to see you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or
+                website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don’t hestiate to contact me<br>
+            </p>
+
+            <a href="#" class="connect">
+                <h3>LET'S CONNECT</h3>
+            </a>
+            <ul class="icons">
+                <li>
+                    <a href="https://twitter.com/christabelleyo" target="blank"><img src="images/Vector.png"
+                            alt="twitter icon" /></a>
+                </li>
+                <li>
+                    <a href="https://github.com/Christabelle143" target="blank"><img src="images/linkedinicon.png"
+                            alt="linkedin icon" /></a>
+                </li>
+                <li>
+                    <a href="https://linkedin.com/youtalactioChristabelle" target="blank"><img src="images/Vector1.png"
+                            alt="meduim icon" /></a>
+                </li>
+                <li>
+                    <a href="https://twitter.com/christabelleyo" target="blank"><img src="images/Vector2.png"
+                            alt="github icon" /></a>
+                </li>
+                <li>
+                    <a href="https://twitter.com/christabelleyo" target="blank"><img src="images/Vector3.png"
+                            alt="angellist icon" /></a>
```

Comment:
- [ ] Please, can you check the Twitter here, and fix the bokken link!

- Your Design:


![okay1](https://user-images.githubusercontent.com/63972670/163853827-43c5fd67-1e53-441a-afa8-6279980d0db6.JPG)

Reply:
- [ ] Kindly make sure to implement the requirement as suggested by the previous reviewer. Fix the Twitter icon as you can see that the Twitter icon is not displayed, make sure that you have given the correct source file name for the icon so that it displays correctly.

Reply:
Good 👍 Its fixed